sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/DataFlowRestClient.cs

```diff
@@ -28,17 +28,13 @@ internal partial class DataFlowRestClient
         /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
         /// <param name="endpoint"> The workspace development endpoint, for example https://myworkspace.dev.azuresynapse.net. </param>
         /// <param name="apiVersion"> Api Version. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="apiVersion"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> is null. </exception>
         public DataFlowRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, string apiVersion = "2019-06-01-preview")
         {
             if (endpoint == null)
             {
                 throw new ArgumentNullException(nameof(endpoint));
             }
-            if (apiVersion == null)
```

Comment:
cc @ShivangiReja 
I think we've removed some checks that we shouldn't have.

Reply:
Fixed here: https://github.com/Azure/autorest.csharp/pull/860